rs/nns/handlers/root/impl/canister/canister.rs

```diff
@@ -111,58 +99,44 @@ async fn canister_status_(canister_id_record: CanisterIdRecord) -> CanisterStatu
     canister_status_response.unwrap()
 }
 
-#[export_name = "canister_update submit_root_proposal_to_upgrade_governance_canister"]
-fn submit_root_proposal_to_upgrade_governance_canister() {
-    over_async(
-        candid,
-        |(expected_governance_wasm_sha, proposal): (
-            serde_bytes::ByteBuf,
-            ChangeCanisterRequest,
-        )| {
-            ic_nns_handler_root::root_proposals::submit_root_proposal_to_upgrade_governance_canister(
-                caller(),
-                expected_governance_wasm_sha.to_vec(),
-                proposal,
-            )
-        },
-    );
+#[update(hidden = true)]
```

Comment:
What's the benefit of hiding some canister functions? 

NCR for this PR

Ditto for `vote_on_root_proposal_to_upgrade_governance_canister`

Reply:
I thin it's b/c they're not considered public APIs, and if we could remove them in favor of a better solution we would.